### PR TITLE
github: update issue templates

### PR DESCRIPTION
**Pull request by creamsoup**

### CODE REVIEW

**Comment by creamsoup** [comment]
i just created some templates for common issue types. any suggestions are welcome (remove, add, edit)

File: .github/ISSUE_TEMPLATE/ask_question.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Ask a question
+about: Asking a question related gRPC-java
+labels: question
+---
+
+Please answer these questions before submitting a bug report.
+
+For non direct gRPC-java related questions, please use [stackoverflow](stackoverflow.com).
```
Comment (sanjaypujare) [inline .github/ISSUE_TEMPLATE/ask_question.md]: "For questions not directly related to gRPC-Java,..."

File: .github/ISSUE_TEMPLATE/ask_question.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Also, if question is not java implementation specific, consider using [grpc.io](https://groups.google.com/forum/#!forum/grpc-io).
```
Comment (sanjaypujare) [inline .github/ISSUE_TEMPLATE/ask_question.md]: java => Java (or even better gRPC-Java)?

File: .github/ISSUE_TEMPLATE/feature_report.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+name: Request a feature
+about: Suggest an idea for gRPC
```
Comment (sanjaypujare) [inline .github/ISSUE_TEMPLATE/feature_report.md]: idea => enhancement

File: .github/ISSUE_TEMPLATE/ask_question.md (hunk 1 above)
Comment (creamsoup) [reply .github/ISSUE_TEMPLATE/ask_question.md]: done.

File: .github/ISSUE_TEMPLATE/ask_question.md (hunk 2 above)
Comment (creamsoup) [reply .github/ISSUE_TEMPLATE/ask_question.md]: done.

File: .github/ISSUE_TEMPLATE/feature_report.md (hunk 3 above)
Comment (creamsoup) [reply .github/ISSUE_TEMPLATE/feature_report.md]: done.

File: .github/ISSUE_TEMPLATE/ask_question.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+name: Ask a question
+about: Asking a question related gRPC-Java
+labels: question
+---
+
+Please answer these questions before submitting a bug report.
```
Comment (ran-su) [inline .github/ISSUE_TEMPLATE/ask_question.md]: maybe "Please read these guidelines before submitting a question."?

Comment (creamsoup) [reply .github/ISSUE_TEMPLATE/ask_question.md]: good catch. actually i removed the line because there is no question in this template.

**Pull request merged**